pyscf/adc/radc.py

```diff
@@ -319,6 +329,8 @@ def analyze(self):
     def compute_dyson_mo(self):
         return self._adc_es.compute_dyson_mo()
 
+    def get_opdm(self):
```

Comment:
Should be consistent with PySCF convention: make_rdm1

Reply:
Renamed to get_opdm to make_rdm1